CommonTools/RecoAlgos/interface/RecoTrackSelectorBase.h

```diff
@@ -19,6 +19,8 @@ class RecoTrackSelectorBase {
     ptMin_(cfg.getParameter<double>("ptMin")),
     minRapidity_(cfg.getParameter<double>("minRapidity")),
     maxRapidity_(cfg.getParameter<double>("maxRapidity")),
+    meanPhi_((cfg.getParameter<double>("minPhi")+cfg.getParameter<double>("maxPhi"))/2.),
+    rangePhi_((cfg.getParameter<double>("maxPhi")-cfg.getParameter<double>("minPhi"))/2.),
```

Comment:
Yes, I think these checks (`minPhi < maxPhi`, `minPhi < π`, `maxPhi > -π`) should be enough to cover all possibilities.

You still leave the possibility of specifying some ranges in multiple ways (e.g. `3/4π..9/4π` vs `-5/4π..1/4π`) but that's not a problem if the code handles both pair of values consistently.